balancer/frontend-v2
# Create a protocol risks page

# Description

A page to outline risks for LPs on the Balancer Protocol.

## Type of change

- [x] New feature (non-breaking change which adds functionality): Enabling anchor links

## How should this be tested?

Go to the new page at /#/risks (also found in the footer) and make sure it displays well across different screen sizes.

View the two links:
- In the footer
- On the Connect Wallet screen.

One important change to test is the scroll behavior across the app. I made a change to the `scrollBehavior` function in `src/plugins/router/index.ts`, in order to enable normal functioning on anchor links which currently don't seem to work across the site. Please review this code.

## Checklist:
